pretix_passbook/passbook.py

```diff
@@ -253,7 +253,10 @@ def generate(self, order_position: OrderPosition) -> Tuple[str, str, str]:
         passfile.description = gettext('Ticket for {event} ({product})').format(event=ev.name, product=ticket)
         passfile.barcode = Barcode(message=order_position.secret, format=BarcodeFormat.QR)
         passfile.barcode.altText = order_position.secret
-        passfile.relevantDate = ev.date_from.astimezone(tz).isoformat()
+        if order.event.settings.show_date_to and ev.date_to and ev.date_to.date() != ev.date_from.date():
```

Comment:
ev.date_from and ev.date_to will always be UTC at this point, in which case a .date() comparison might yield a useless result for an Event in australia. You should never call .date() without calling .astimezone() first :)

Reply:
Thanks a lot. I fixed it.